website/versioned_docs/version-0.61/rnm-dependencies.md

```diff
@@ -4,19 +4,19 @@ title: System Requirements
 original_id: rnm-dependencies
 ---
 
-You can run React Native for macOS apps on Mac devices with versions [Mojave (10.14.6)](https://apps.apple.com/us/app/macos-mojave/id1398502828?ls=1&mt=12) or newer.
+You can run React Native for macOS apps on Mac devices with versions [Mojave (10.14.6)](https://support.apple.com/en-us/HT209149#macos10146) or newer.
 
 To develop React Native for macOS apps, you will need the following:
 
 ### macOS Development Dependencies
 - A Mac device is required to build projects with native code for macOS.
-- Install [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) version 9.4 or newer.
+- Install [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) version 11.3.1 or newer.
 - Ensure to install Xcode Command Line Tools. Open Xcode, then choose "Preferences..." from the Xcode menu. Go to the Locations panel and install the tools by selecting the most recent version in the Command Line Tools dropdown.
 - Install [CocoaPods](https://guides.cocoapods.org/using/getting-started.html)
 
 ### React Native Development Dependencies
 - Install the [standard React Native dependencies](https://reactnative.dev/docs/getting-started#node-python2-jdk)
```

Comment:
Think it would be better to link to this page: https://reactnative.dev/docs/environment-setup.   

But even that reactnative.dev page defaults to the expo setup and you have to click a tab to show the "React Native CLI Quickstart".   From there the "Development OS: macOS" > "Target OS: iOS" dependencies are what the dev needs to install to be ready for macOS development.

Maybe the button can say "- Install the standard React Native dependencies (click "React Native CLI Quickstart" from that page)"

We had a bug bash on the flow yesterday and installing the deps was defintely a pain point.   I'll suggest more explicit steps below too paraphrasing from the React Native CLI Quickstart.

Reply:
Suggestion: maybe at the very top of this page we can say something like:

"The development dependencies for React Native for macOS are identical to the development dependencies for React Native for iOS.   You can follow the steps at [Setting up the development environment](https://reactnative.dev/docs/environment-setup) in the **"React Native CLI Quickstart"** section, or follow the paraphrased steps below.